15_paste_from_register_in_insert_mode.md

```diff
@@ -10,7 +10,7 @@ Read Drew Neil's
 Copy and paste the page title here:
 
 `yt,` - yanks the text 'Practical Vim
-`jA`  - jumps to the end of the second line
+`jA ` - jumps to the end of the second line and add a space
```

Comment:
Can you plz follow the convention and use `adds` instead of `add`?

Reply:
done.